packages/vc-verification/src/issuer-credential-resolver.ts

```diff
@@ -0,0 +1,23 @@
+import { StorageSettings } from './models';
+import { DidStore } from '@ew-did-registry/did-ipfs-store';
+import { IDidStore } from '@ew-did-registry/did-store-interface';
+
+export class CredentialResolver {
+  private _ipfsStore: IDidStore;
+  private _ipfsUrl: string;
+
+  constructor(storageSettings: StorageSettings) {
+    this._ipfsUrl = storageSettings.ipfsApi;
+    this._ipfsStore = new DidStore(this._ipfsUrl);
+  }
+
+  /**
+   *
+   * @param serviceEndpoint
+   * @returns {string}
+   */
+  async getIssuerCredential(serviceEndpoint: string): Promise<string> {
```

Comment:
I think it would be good to make this resolution more robust by copying the code from passport-did-auth. It has some code to deal with edge cases in legacy credentials https://github.com/energywebfoundation/passport-did-auth/blob/develop/lib/LoginStrategy.ts#L267
What do you think?